sp20/syllabus.yml

```diff
@@ -77,9 +81,9 @@ meeting03:
       youtube: ''
     kaggle:
       datasets: []
-      competitions: []
+      competitions: ["ucfai-core-sp20-cnns"]
```

Comment:
@brandons209 said you need it – you actually don't. 😅 that's my b on explaining.